From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 11 Oct 2011 09:37:27 -0600
Patch-mainline: v3.2-rc1
Subject: [PATCH] hv: hyperv.h: remove unneeded forward declarations of structures
Git-commit: 2726f95e0b6c850b6162f287f2f83d9db37decd7

This file was created by mushing different .h files together and it
shows.  This change removes some unneeded forward declarations.

Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 include/linux/hyperv.h |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
index f2bf0a5..ae865a1 100644
--- a/include/linux/hyperv.h
+++ b/include/linux/hyperv.h
@@ -693,9 +693,6 @@ extern void vmbus_get_debug_info(struct vmbus_channel *channel,
 
 extern void vmbus_ontimer(unsigned long data);
 
-struct hv_driver;
-struct hv_device;
-
 struct hv_dev_port_info {
 	u32 int_mask;
 	u32 read_idx;
-- 
1.6.0.2


